mycli/packages/special/utils.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+
+def handle_cd_command(arg):
+    """Handles a `cd` shell command by calling python's os.chdir."""
+    CD_CMD = 'cd'
+    command = arg.strip()
+    directory = ''
+    error = False
+
+    tokens = arg.split(CD_CMD + ' ')
+    directory = tokens[-1]
+
+    try:
+        os.chdir(directory)
+        output = subprocess.check_output('pwd', stderr=subprocess.STDOUT, shell=True)
```

Comment:
`subprocess.check_output()` was only introduced in Python 2.7. We've made a conscious decision to support Python 2.6 and above. So we can't use check_output(). But using `subprocess.call()` achieves the same results.